docs/_layouts/default.html

```diff
@@ -1,21 +1,35 @@
-<!DOCTYPE html>
-<html lang="{{ page.lang | default: site.lang | default: "en" }}">
-
-  {% include head.html %}
-
-  <body>
+<!--
+ __  __                         _____                   
+|  \/  |                       / ____|                  
+| \  / |_   ____   ___ __ ___ | |     _ __ ___  ___ ___ 
+| |\/| \ \ / /\ \ / / '_ ` _ \| |    | '__/ _ \/ __/ __|
+| |  | |\ V /  \ V /| | | | | | |____| | | (_) \__ \__ \
+|_|  |_| \_/    \_/ |_| |_| |_|\_____|_|  \___/|___/___/
+========================================================
+                                    © 2017 MvvmCross.com
+-->
 
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: en }}">
+<!-- head -->
+{% include html-head.html %}
+<body>
+    <!-- header -->
     {% include header.html %}
-
-    <main class="page-content" aria-label="Content">
-      <div class="wrapper">
+    <!-- main content -->
+    <main class="container" role="main">
         {{ content }}
-      </div>
+        <div id="back-to-top" role="button" aria-label="Back to top">
+            <i class="fa fa-angle-double-up" aria-hidden="true"></i>
+        </div>
     </main>
-
+    <!-- footer -->
     {% include footer.html %}
-    {% include google-analytics.html %}
-    {% include deeplink.html %}
-  </body>
-
-</html>
+    <!-- scripts -->
+    {% include scripts.html %}
+    <!-- google analytics -->
+    {% if site.github.environment == "dotcom" %}
```

Comment:
This comparision on "dotcom" will make sure that we only get analytics on mvvmcross.com?